### PR TITLE
Bump Nav Native to v26.1 and Common SDK to 9.1.0 and changelog for 1.2.0-alpha.4

### PR DESCRIPTION
**Pull request by LukasPaczos**

### CODE REVIEW

**Review by Guardiola31337 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,23 @@
...
+Visit our [1.0.0 migration guide](https://github.com/mapbox/mapbox-navigation-android/wiki/1.0-Navigation-SDK-Migration-Guide) and the [documentation pages](http://docs.mapbox.com/android/navigation/overview) for more information.
+
+### Changelog
+#### Improvements
+* Removed some synchronized disk access events from upstream library. [#3675](https://github.com/mapbox/mapbox-navigation-android/pull/3675)
+* Improved exceptions handling when toggling native history collection. [#3705](https://github.com/mapbox/mapbox-navigation-android/pull/3705)
+
+#### Bug fixes
+* Fixed additional edge cases that occured when trying to infer the route line's z-ordering on the map if not explicitly specified. [#3687](https://github.com/mapbox/mapbox-navigation-android/pull/3687) [#3702](https://github.com/mapbox/mapbox-navigation-android/pull/3702)
+* Fixed an issue that would make `off-route` and `invalid-route` inconsistent when there were no available road edges to map-match to. The state will now report `off-route` in those situations. [#3705](https://github.com/mapbox/mapbox-navigation-android/pull/3705)
```
Comment (Guardiola31337) [inline CHANGELOG.md:15]: Wrong ticket number and link

Comment (Guardiola31337) [reply CHANGELOG.md:15]: It's pointing to this PR 😅

Comment (LukasPaczos) [reply CHANGELOG.md:15]: I think that's correct, it was fixed in 26.1.

Comment (Guardiola31337) [reply CHANGELOG.md:15]: Ohh I see 👍

Comment (LukasPaczos) [reply CHANGELOG.md:15]: Well :shrug:

**Pull request merged**